interfaces/portal/src/app/pages/program-registration-debit-cards/components/link-card-on-site-dialog/link-card-dialog.component.html

```diff
@@ -0,0 +1,141 @@
+<p-dialog
+  [visible]="dialogVisible()"
+  modal
+  closable
+  dismissableMask
+  [style]="{ width: '632px' }"
+>
+  @if (linkCardDialogState() === linkCardDialogStates.linking) {
+    <ng-template pTemplate="header">
+      <div class="flex items-center gap-2">
+        <i class="pi pi-link"></i>
+        <h3 i18n>{{ dialogLabels().header }}</h3>
+      </div>
+    </ng-template>
+    <div class="flex flex-col gap-2">
+      @if (currentTokenCode()) {
+        <p i18n>
+          You're about to replace debit card number
+          {{ currentTokenCode() }}.
+        </p>
+      }
+      <p i18n>
+        Scan the barcode in the back of the card you are handing over to the
+        registration with a scanner, or manually fill in the 19 digit serial
+        number found below the barcode.
+      </p>
+      <div class="py-1"></div>
+      <img
+        src="assets/intersolve-visa-card.svg"
+        height="200"
+        width="400"
+        alt=""
+        aria-hidden="false"
+        class="mx-auto"
+      />
+      <label
+        class="block w-full"
+        for="serial-number-input"
+      >
+        <span
+          class="mb-1 block font-bold"
+          i18n
+          >Serial number</span
+        >
+        <p-input-mask
```

Comment:
If there's already an explicit connection between the `label` and the `input`, the "implicit-connection by wrapping it" wouldn't be necessary, right?

Also, doesn't the `p-input-mask`-component provide a `label`-property? So that it works the same as other "text-input"-components?